pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/catalog/PmdCatalogRule.java

```diff
@@ -14,6 +17,23 @@ public class PmdCatalogRule {
 	public static final String ATTR_MESSAGE = "message";
 	public static final String ATTR_DESCRIPTION = "description";
 
+    public static final String ATTR_LANGUAGE = "language";
+    public static final String ATTR_CLASS = "class";
+    public static final String ATTR_REF = "ref";
+
+    /**
+     * Regex that matches the naming convention used by PMD's JARs.
+     * Ordinarily, this would be less hardcoded, but PMD's not releasing new versions of 6,
+     * so we can count on the JAR remaining constant until PMD 7 releases, at which point this
+     * code will probably be obsolete.
+     */
+    private static final Pattern STANDARD_JAR_PATTERN = Pattern.compile("pmd-(apex|java|javascript|visualforce|xml)-6\\.55\\.0\\.jar", Pattern.CASE_INSENSITIVE);
```

Comment:
See the comment. Ordinarily I wouldn't hardcode this, but we know for a fact that the value won't change because PMD 6 isn't releasing new versions anymore.